frankDoc/src/main/java/nl/nn/adapterframework/frankdoc/doclet/FrankMethodDoclet.java

```diff
@@ -157,7 +157,12 @@ public String getJavaDocIncludingInherited() throws FrankDocException {
 		MethodDoc overriddenMethodDoc = method.overriddenMethod();
 		if(overriddenMethodDoc != null) {
 			FrankMethodDoclet overriddenMethod = (FrankMethodDoclet) declaringClass.recursivelyFindFrankMethod(overriddenMethodDoc);
-			return overriddenMethod.searchExcludingImplementedInterfaces(getter);
+			if(overriddenMethod != null) {
+				return overriddenMethod.searchExcludingImplementedInterfaces(getter);
+			} else {
+				log.warn("Method {}.{} overrides {}.{}, but that is not detected by the Frank!Doc wrapper API",
+						getDeclaringClass().getName(), getName(), overriddenMethodDoc.containingClass().name(), overriddenMethodDoc.name());
```

Comment:
Ik heb dit door de telefoon uitgelegd aan Gerrit. Ik begrijp niet precies wat hier gebeurt, dus het is niet duidelijk wat deze warning betekent. Dit moet opgelost worden voordat deze pull request gemerged kan worden. Daarom ga ik deze pull request weer draft maken.

Reply:
Toen ik dit ging uitzoeken ontdekte ik een fout in mijn unit tests. Je kan in een doclet configureren welke access modifiers meedoen. In de unit tests nam ik alle access modifiers mee, terwijl de echte Frank!Doc doclet alleen public access meenam. Ik heb dit gecorrigeerd.

Reply:
De unit tests worden gefixed in frankDoc/src/test/java/nl/nn/adapterframework/frankdoc/doclet/classdocs/EasyDoclet.java. Er vielen daarna testen om en er zijn daarom correcties toegevoegd.